horizontal_pod_autoscaler.tf

```diff
@@ -0,0 +1,25 @@
+# Create a horizontal pod autoscaler for the service when autoscaling is enabled
+resource "kubernetes_horizontal_pod_autoscaler" "ambassador" {
```

Comment:
Not sure I understand, are we not just turning on autoscaling for ambassador? I think it would feel cleaner to have it enabled with min/max set to 1/5.

Yes will probably have to add the ignore replicas (like kinesis-autoscaler?)

Reply:
`ignore_changes` can't be set dynamically in terraform, so it's not very easy to get the `autoscaling_enable` var to work without creating a whole separate deployment with the `ignore_changes` directive in it. I was originally hoping to be able to make this change in a way that was backwards compatible, but I think it'll be a lot easier if I just enable autoscaling by default.

Have added another commit which does the above, and also sets the default min/max pods to 3 and 6 respectively (that way we'll be able to get one per availability zone).